src/test/java/org/dice_research/opal/civet/metrics/ProviderIdentityMetricTest.java

```diff
@@ -34,45 +42,55 @@
 	 * accessURL can be used as landingPage is distributions are accessed 
 	 * only through accessURL.
 	 */
-	private static final String TestCase_3stars = "TestCaseForPI_Access_URL_3Stars__synthetic.ttl";
+	private static final String TestCase_3stars = "TestCaseForPI_3Stars__synthetic.ttl";
+	private static final String TEST_dataset_3stars = "http://projekt-opal.de/dataset/https___ckan_govdata_de_34a3e19c_323f_474a_b21b_b03be924093a";
 	
 	
 	/*
 	 * No publisher information at all
 	 */
 	private static final String TestCase_0stars = "TestCaseForPI_0Stars__synthetic.ttl";
-
-	private static final String TEST_dataset_5stars = "http://projekt-opal.de/dataset/https___ckan_govdata_de_a642d7e5_bf1c_57f9_89df_cdad67c7c0fc";
-	private static final String TEST_dataset_3stars = "http://projekt-opal.de/dataset/https___europeandataportal_eu_set_data_c0e483fd_9d29_328f_9b83_9eb950c9f022";
-	private static final String TEST_dataset_0stars = "http://projekt-opal.de/dataset/https___europeandataportal_eu_set_data_c0e483fd_9d29_328f_9b83_9eb950c9f022";
+	private static final String TEST_dataset_0stars = "http://projekt-opal.de/dataset/_mcloudde_vieljhrlicherasterdesmittlerenvegetationsbeginnsindeutschland";
 
 	@Before
 	public void setUp() throws Exception {
 		testdata = new TestData();
 	}
-
+	
 	@Test
-	public void TestCase1() throws Exception {
+	public void testCase1() throws Exception {
 
 		ProviderIdentityMetric metric = new ProviderIdentityMetric();
-		Integer stars = metric.compute(testdata.getModel(TestCase_5stars), TEST_dataset_5stars);
-		Assert.assertEquals("Provider Identity Test: Test Case 5 stars", 5, stars.intValue());
+		Integer stars = metric.compute(testdata.getModel(TestCase_0stars), TEST_dataset_0stars);
+		Assert.assertEquals("Provider Identity Test: Test Case 0 stars", 0, stars.intValue());
 	}
 	
+	
 	@Test
-	public void TestCase2() throws Exception {
+	public void testCase2() throws Exception {
 
 		ProviderIdentityMetric metric = new ProviderIdentityMetric();
 		Integer stars = metric.compute(testdata.getModel(TestCase_3stars), TEST_dataset_3stars);
 		Assert.assertEquals("Provider Identity Test: Test Case 3 stars", 3, stars.intValue());
 	}
+
 	
 	@Test
-	public void TestCase3() throws Exception {
+	public void testCase3() throws Exception {
 
 		ProviderIdentityMetric metric = new ProviderIdentityMetric();
-		Integer stars = metric.compute(testdata.getModel(TestCase_0stars), TEST_dataset_0stars);
-		Assert.assertEquals("Provider Identity Test: Test Case 0 stars", 0, stars.intValue());
+		Integer stars = metric.compute(testdata.getModel(TestCase_4stars), TEST_dataset_4stars);
+		Assert.assertEquals("Provider Identity Test: Test Case 4 stars", 4, stars.intValue());
+	}
+	
+	
+	@Test
+	public void testCase4() throws Exception {
```

Comment:
There is something wrong here in the test cases file. 
1. Because If I am running the test cases individually there is a perfect result. 
2. If I am running all the test cases then testcase4() does not gives a perfect result. 
3. Also If I comment testcase1() and testcase3(), there is some another result. 

I have tried a lot to find the issue, but unfortunately could not get to the actual error by which it is getting. 
Could Adrian please give your views on this case?
I think J-unit testing was under Amit so maybe he could help us?
Looking forward for the views, Thanks.

Reply:
Yes, I already invested a lot of time to find out why they are failing when run together. They are all passing when running individually.

Reply:
@adibaba

Reply:
that's simple.
to both of you: please check the meaning of the word "static".
afterwards, edit the metric class. hint: edit (at least) 3 variables and 1 method.

Reply:
All test cases are passing now :D